packages/firestore/src/local/local_store.ts

```diff
@@ -192,6 +200,11 @@ export class LocalStore {
     this.queryEngine.setLocalDocumentsView(this.localDocuments);
   }
 
+  /** Starts the LocalStore. */
+  start(): Promise<void> {
+    return this.synchronizeLastDocumentChangeReadTime();
```

Comment:
This is equivalent to the previous functionality in RemoteDocumentStore.start().